fwdpy11/_functions/import_demes.py

```diff
@@ -517,22 +517,19 @@ def _process_epoch(
     if e.start_time != math.inf:
         assert size_history.deme_exists_at(idmap[deme_id], when + 1)
 
+        start_size = int(np.rint(e.start_size))
```

Comment:
Here is a good place to ensure that:

* end_size > start_size (???)
* the duration is > 0 when converted to integer?

Reply:
Why do we need to check that end_size > start_size?

But yes, we need to check that duration is 1 generation or greater, I agree.

Reply:
My mistake -- I meant start/end time, meaning epoch length > 0.  We'll take care of that in another PR so that the git history reflects both this change and the check on epoch lengths.